src/components/WebexActivity/WebexActivity.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {format, isToday, isSameWeek, isYesterday} from 'date-fns';
+
+import WebexAvatar from '../WebexAvatar/WebexAvatar';
+import {useActivity, usePerson} from '../hooks';
+
+import './WebexActivity.scss';
+
+/**
+ * Returns a formatted timestamp based on the given date's offset from the current time.
+ *
+ * @param {Date} timestamp A string value representing a date, specified in a format recognized by the Date.parse()
+ * @returns {string} formatMessageDate
+ */
+export function renderedDate(timestamp) {
+  let formatMessageDate;
+
+  if (isToday(timestamp)) {
+    // 12:00 PM
+    formatMessageDate = format(timestamp, 'p');
+  } else if (isYesterday(timestamp)) {
+    // Yesterday 12:00 PM
+    formatMessageDate = `Yesterday ${format(timestamp, 'p')}`;
+  } else if (isSameWeek(timestamp, new Date())) {
+    // Monday 12:00 PM
+    formatMessageDate = format(timestamp, 'iiii p');
+  } else {
+    // 1/1/2020 12:00 PM
+    formatMessageDate = format(timestamp, 'P p');
+  }
+
+  return formatMessageDate;
+}
+
+export function Header(props) {
+  const {personID, adapter, created} = props;
+  const {displayName} = usePerson(personID, adapter);
+
+  return (
+    <div className="activity-header">
+      <WebexAvatar personID={personID} adapter={adapter} />
+      <div className="activity-author">
+        <span>{displayName}</span>
+        <span className="activity-timestamp">{renderedDate(new Date(created))}</span>
+      </div>
+    </div>
+  );
+}
+
+Header.propTypes = {
+  personID: PropTypes.string.isRequired,
+  adapter: PropTypes.object.isRequired,
+  created: PropTypes.string.isRequired,
+};
+
+export default function WebexActivity(props) {
+  const {activityID, adapters} = props;
+  const {created, displayHeader, ID, personID, text} = useActivity(activityID, adapters.activitiesAdapter);
+  const header = displayHeader ? (
+    <Header personID={personID} adapter={adapters.peopleAdapter} created={created} />
+  ) : null;
+
+  return (
+    <div className="activity" key={ID}>
+      {header}
+      <div className="activity-content">{text}</div>
+    </div>
+  );
+}
+
+WebexActivity.propTypes = {
+  activityID: PropTypes.string.isRequired,
+  adapters: PropTypes.exact({
+    activitiesAdapter: PropTypes.object.isRequired,
+    peopleAdapter: PropTypes.object.isRequired,
+  }),
```

Comment:
Are you able to add `.isRequired` here and remove the `defaultProps` definition?

Reply:
yeah that works